docs/configuration.rst

```diff
@@ -69,4 +76,12 @@ In your ``.behaverc`` file, you can put
 
 .. |keepdb docs| replace:: More information about ``--keepdb``
 .. _keepdb docs: https://docs.djangoproject.com/en/stable/topics/testing/overview/#the-test-database
-.. _behave docs: https://behave.readthedocs.io/en/latest/behave.html#configuration-files
+.. _Django test runner: https://docs.djangoproject.com/en/stable/ref/settings/#test-runner
+.. _Django's default test runner: https://github.com/django/django/blob/stable/4.0.x/django/test/runner.py#L555-L582
+.. |BehaviorDrivenTestRunner| replace:: ``behave_django.runner.BehaviorDrivenTestRunner``
+.. _BehaviorDrivenTestRunner: https://github.com/behave/behave-django/blob/main/behave_django/runner.py#L11-L15
```

Comment:
This could be problematic no? If new lines are added above it, this whole section goes kaput!
(same for line 83)

Reply:
You're too fast for me @bittner  :P

Reply:
I'm not sure I understand. Do you refer to the anchor? Or the source code the link makes you land in?

Reply:
The line numbers. 
On line 82 you're referring to Django's `main` and `main` will undoubtedly change.

Reply:
Aha, with the tag in the URL this would be more reliable? e.g. https://github.com/behave/behave-django/blob/1.4.0/behave_django/runner.py#L9-L13

Reply:
Yup, or the hash... to be SUPER safe

Reply:
Granted. This will not change in the next week... so it's safe right now ;)

Reply:
I fixed that with commit 91a26b631. Sorry for merging prematurely.

P.S.: Have you taken a look at #129? I could add you as a reviewer in PRs then. :slightly_smiling_face:

Reply:
Yup. That works :D